docs/api.md

```diff
@@ -138,27 +138,34 @@ Neutrino({
 ### `options.mains`
 
 Set the main entry points for the application. If the option is not set, Neutrino defaults it to:
```

Comment:
There is a lot of pre-existing duplication between `api.md`, `creating-presets.md` and `customization.md` that is out of scope for this PR, but something we should figure out how to avoid at some point.